### ISSUE TITLE
Tag not rendering if the only tag is a number

### ISSUE THREAD

**Issue by lukewduncan** [yairEO/tagify#341]
## Prerequisites

- [x] I am running the latest version. We were running 2.18.4 - we thought bumping to the latest version might solve the problem but it did not.
- [x] I checked the documentation and found no answer
- [x] I checked to make sure that this issue has not already been filed

### Code

JS
```
var input = document.querySelector('input.tagify--event_groups'),
        tagify = new Tagify(input, {
          whitelist: "example",
          backspace: false,
          maxTags: 3,
          dropdown : {
            enabled: 1,
          }
        });
```

HTML

This is not working
```
<input class="tagify--event_groups" " id="tagify-input" name="event_group_list" placeholder="Enter your choices here..." type="text" value="2019">
```

This is working
```
<input class="tagify--event_groups" " id="tagify-input" name="event_group_list" placeholder="Enter your choices here..." type="text" value="2019, boom">
```

This is working
```
<input class="tagify--event_groups" " id="tagify-input" name="event_group_list" placeholder="Enter your choices here..." type="text" value="boom">
```

### Explanation

- Was expecting the tag "2019" to render. It does not.
- The input is empty, but we are still able to enter tags. When we enter any number as the first tag and don't input any more tags, refresh the page, the tag does not render. If we enter a word and a number it works fine. If we enter just a single word it works fine.
- Before we version bumped to the lastest - we were receiving this console.log:
`[addTags] no tags to add: 2019`

Tried looking into the code myself but can't see any obvious reasons why this would not be working. Maybe i'm not understanding something?

Thanks in advance. Love the plugin regardless.

**Comment by yairEO** [comment]
Thanks for reporting, I will look into it tomorrow and let you know

**Comment by lukewduncan** [comment]
Thanks for your help in advance! @yairEO

**Comment by yairEO** [comment]
sorry I went on a vacation and haven't gotten to it but today I will have time

**Comment by yairEO** [comment]
Fixed and will be pushed today.

The reason was in the `addTags` method, where there was an inappropriate check for `length` at the top of the method, which failed for type *Number*, since in JS `length` is `undefined` for type `Number`. Changes to:

    if( !tagsItems || tagsItems.length == 0 )

**Comment by lukewduncan** [comment]
Hey @yairEO - really appreciate it! This project is awesome. My team would like to show our appreciation! Do you have a place where we can send you money for a coffee or donation?

**Comment by yairEO** [comment]
## Thank you!

paypal: `vsync.design@gmail.com `

bitcoin: `37idub7iLDFdqqz3LpQW1KL8cjyXYVrzJo`

**Comment by yairEO** [comment]
Fixed on version `2.31.6`, now published
